tools/browse_dataset.py

```diff
@@ -27,7 +29,7 @@ def parse_args():
     parser.add_argument(
         '--show-interval',
         type=float,
-        default=1,
```

Comment:
Is there any specific reason to change the default behavior?

Reply:
Because the window closes too fast,  the display effect is not good